.travis.yml

```diff
@@ -1,20 +1,18 @@
 language: node_js
 node_js:
   - '8.9.4'
-before_script:
-  - node common/scripts/ci-rush.js
 script:
   - set -e
-  - echo 'Checking change file...' && echo -en 'travis_fold:start:check\\r'
+  - echo 'Checking change file...' && echo -en 'travis_fold:start:change\\r'
   - git fetch origin master:refs/remotes/origin/master -a
-  - node common/temp/ci-rush/node_modules/@microsoft/rush/lib/start.js change -v
+  - node common/scripts/install-run-rush.js rush change -v
```

Comment:
>rush [](start = 44, length = 4)

Let's eliminate the command for this one.  If we want to run rushx, we could just provide an install-run-rushx.js command, since very little code would get copied